trillian_log_api.proto

```diff
@@ -99,6 +99,25 @@ message QueueLeavesResponse {
     repeated QueuedLogLeaf queued_leaves = 2;
 }
 
+message AddSequencedLeafRequest {
+    int64 log_id = 1;
+    LogLeaf leaf = 2;
+}
+
+message AddSequencedLeafResponse {
+    google.rpc.Status status = 2;
```

Comment:
Not sure you need these `status` fields (or at least I don't think the other method responses contain them - you'd normally just return a status.Errorf(code, ...) as the error being returned from the RPC method.

Reply:
Yeah, I also wasn't sure about this list. Are use suggesting to return an empty proto instead (since the `error` returned from `status.Errorf(...)` seems to be handled outside of the proto)?

Consider, however, that I want the API to help distinguish the following cases:
- All entries are stored successfully.
- Some entry is not stored because the corresponding index is occupied by:
   - a different entry, which is bad. Error.
   - a duplicate entry, which is okay. Warning (or just OK?).
- Other arbitrary error.

Do you think I can put all these into the `error`?

Reply:
For a single leaf request, I think you can use an empty proto and return error directly, e.g. `FailedPrecondidtion` for different entry, `AlreadyExists` for identical entry, etc. - check out the [gRPC codes](https://godoc.org/google.golang.org/grpc/codes).

For the batch API I agree you'll want a status returned per leaf requested.